src/Spryker/Zed/SynchronizationBehavior/Persistence/Propel/Behavior/SynchronizationBehavior.php

```diff
@@ -1029,7 +1064,7 @@ protected function setGeneratedAliasKeys()
             ];
         }
     }
-    \$aliasKeys = json_encode(array_unique(\$aliasKeys));
+    \$aliasKeys = json_encode(\$aliasKeys);
```

Comment:
Why do we need this change?

Reply:
This was causing an issue with multiple mappings because array_unique tries to cast the values of an array argument to string for comparison without a special flag passed as the second parameter. I could have passed `SORT_REGULAR` as the second argument to the array_unique function to resolve this, but actually it seems that we don't really need this function call - there cannot be to elements with the same values here.